pisek/env/config_hierarchy.py

```diff
@@ -16,27 +16,47 @@
 
 from collections import defaultdict
 from configparser import ConfigParser
+from dataclasses import dataclass
 from importlib.resources import files
 import os
 import re
+from typing import Optional, Any
 
 from pisek.utils.text import tab
 from pisek.env.config_errors import TaskConfigError
 from pisek.env.update_config import update_config
 
-DEFAULTS_CONFIG = str(files("pisek").joinpath("defaults-config"))
+DEFAULTS_CONFIG = str(files("pisek").joinpath("env/global-defaults"))
 CONFIG_FILENAME = "config"
 
 
+@dataclass
+class ConfigValue:
+    value: Any
+    config: str
+    section: str
+    key: Optional[str]
+    internal: bool = False
+
+    def location(self) -> str:
+        text = "Internal pisek value" if self.internal else f"Config {self.config}"
```

Comment:
I find it a bit confusing how the message just starts with "Config config".
Could we maybe force it to say "Config ./config"?